### PR TITLE
Updated readme.md

### LINKED ISSUE #29: Beef up README.md
We have some solid documentation, but I think it would be best to add some additional instructions to `README.md`.
Quickstart instructions, especially of the **Docker** variety, would encourage more people to spin up the project and contribute to it.

### PR DESCRIPTION
**Pull request by OlaPom**
Hi again!
As I explained already on Discord, these are my suggestions to the README file:
- installation instructions copied from Install, Setup, and Run with Docker/ manually sections of the website
- short description of the tool "About DRC"
- logo and screenshot (I've seen that there are more screenshots in your repo so it can be changed)

Let me know if there is anything you want to change.

### CODE REVIEW

**Comment by OlaPom** [comment]
Issue #29

File: README.md [hunk 1]
```diff
@@ -7,12 +9,131 @@ I heard you liked IRC & Discord so I put some IRC in your Discord.
...
+
+- `app.allowedSpeakersRoleId` or `app.allowedSpeakers`
+- `discord.botId`
+- `discord.guildId`
+- `discord.token`
+- `irc.quitMsgChanId`
+
+### Using private message channels
+
+If you create a category with a name that matches [this logic](https://github.com/edfletcher/drc/blob/main/discord.js#L347)\* - such as “Private Messages”, “privmsgs”, or “PMs”) - DRC will automatically create a channel in this category for each private message received (a la traditional IRC client’s “query window” feature).
```
Comment (edfletcher) [inline README.md]:
```suggestion
If you create a category with a name that matches [this logic](https://github.com/edfletcher/drc/blob/d4d7e8811eeb70c0fd37edf94d006744db1e61a4/discord.js#L629)\* - such as “Private Messages”, “privmsgs”, or “PMs”) - DRC will automatically create a channel in this category for each private message received (a la traditional IRC client’s “query window” feature).
```

(Fixing a mistake of mine in the source documentation, not using a permalink means that the line reference moves over time and becomes incorrect. This fixes that.)

File: README.md [hunk 2]
```diff
@@ -7,12 +9,131 @@ I heard you liked IRC & Discord so I put some IRC in your Discord.
...
+You may set `DEBUG=1` in the environment to enable more-verbose logging from any of the daemons.
 
 ## Contributors
 
-* [EdFletcher](https://github.com/edfletcher)
-* [antishok](https://github.com/antishok)
-* [CovertDuck](https://github.com/CovertDuck)
+- [EdFletcher](https://github.com/edfletcher)
+- [antishok](https://github.com/antishok)
+- [CovertDuck](https://github.com/CovertDuck)
```
Comment (edfletcher) [inline README.md:139]:
```suggestion
- [CovertDuck](https://github.com/CovertDuck)
- [OlaPom](https://github.com/OlaPom)
```

(You are a contributor now! 🙇 😄 )

**Review by edfletcher — APPROVED** [review]
Only two comments: one to fix a mistake of _mine_ from the source documentation, and another to make sure you are added to the "Contributors" list appropriately! Welcome to the team and thank you so much for this truly excellent work.

LGTM ship it! 🚀

**Comment by OlaPom** [comment]
This is done. Thanks for adding me to the list :) If you have any other ideas about the documentation (or if I have any), I'd be happy to help.

**Comment by edfletcher** [comment]
Wooo hoo, merged! 🚀 Thank you again @OlaPom, and yes _please_ if you come across anything else you'd like to improve - either here or [on the main site](https://github.com/edfletcher/discordrc.com) - feel free to submit more PRs. I really do appreciate your contributions and am ecstatic to have you as part of the team! 🙇

**Pull request merged**